packages/devtools-launchpad/src/components/sidebar.scss

```diff
@@ -0,0 +1,84 @@
+$sidebar-width: 200px;
```

Comment:
can we remove `sidebar.css`? Can we rename `sidebar.scss` to `sidebar.css`? It'd be nice not to differentiate between the two

Reply:
Removed the sidebar.css. Don't think it's common to name give a .css suffix to a sass file...